libgit2-sys/lib.rs

```diff
@@ -600,6 +603,7 @@ pub enum git_ref_t {
 }
 pub use git_ref_t::*;
 
+enum_from_primitive!{
```

Comment:
I'd prefer to not pick up this dependency for now, I think that it may just be best to remove this altogether and make it a raw `i32` as I think that's basically what it's later treated as anyway.